src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -622,8 +623,9 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
             withEntity(workspaceContext, entityType, entityName, txn) { entity =>
               withMethod(workspaceContext, methodConfig.methodRepoMethod.methodNamespace, methodConfig.methodRepoMethod.methodName, methodConfig.methodRepoMethod.methodVersion, authCookie) { method =>
                 withWdl(method) { wdl =>
-                  // TODO should we return OK even if there are validation errors?
-                  RequestComplete(StatusCodes.OK, MethodConfigResolver.getValidationErrors(workspaceContext, methodConfig, entity, wdl, txn))
+                  val inputs = MethodConfigResolver.resolveInputs(workspaceContext, methodConfig, entity, wdl, txn)
+                  val idation = executionServiceDAO.validateWorkflow(wdl, MethodConfigResolver.propertiesToWdlInputs(inputs), authCookie)
```

Comment:
what does this validate that has not already been validated? Why do we need a web service call to cromwell to validate a workflow?